src/components/search/form.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+class Form extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            text:""
+        }
+    }
+
+    render() {
+        const onClick = this.props.onClick;
+        return (
+            <div>
+                <input type="text" ref={node => {
```

Comment:
You do not need to use a ref here. Look at controlled and uncontrolled components.